docs/webhooks/index.html

```diff
@@ -25,17 +27,51 @@ <h1 class="header-title">CodeSignal Webhook API</h1>
         </div>
         <div class="main-content">
             <div class="section">
+                <div class="subsection">
+                    <h2>Table of Contents</h2>
+                    <ul>
+                        <li>Introduction</li>
+                        <ul>
+                            <li><a href="#whatwebhook">What is a webhook?</a></li>
+                            <li><a href="#whatinfo">What information is needed to create a webhook?</a></li>
+                            <li><a href="#howcreate">How do I create a webhook?</a></li>
+                        </ul>
+                        <li>Usage</li>
+                        <ul>
+                            <li><a href="#auth">Authentication</a></li>
+                            <li><a href="#eventhandling">Event handling</a></li>
+                            <li><a href="#retry">Retry policy</a></li>
+                            <li><a href="#payloads">Webhook payloads</a></li>
+                        </ul>
+                        <li>Events for CodeSignal Test:</li>
+                        <ul>
+                            <li><a href="#testworkflow">Test Webhooks Process Flow</a></li>
```

Comment:
For this item and the other one (`Certify Webhooks Process Flow`), just for visual consistency, I think it'd be lovely to use sentence case instead of title case. Right now they stand out a little because they're the only ToC items in title case:
![Screen Shot 2021-03-22 at 2 14 28 PM](https://user-images.githubusercontent.com/747378/112059322-f15fde80-8b18-11eb-9a50-2dde21174680.png)

i.e., maybe just `Test webhooks process flow` and `Certify webhooks process flow`